packages/lib/shared/constants/ui.constants.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * UI-related constants
+ */
+
+// Dash symbol used for zero balances and empty values
+export const ZERO_VALUE_DASH = '-'
+
+// Display labels for small amounts
+export const SMALL_AMOUNT_LABEL = '<0.001'
+export const SMALL_PERCENTAGE_LABEL = '<0.01%'
+
+// Thresholds for display formatting
+export const AMOUNT_LOWER_THRESHOLD = 0.001
+export const PERCENTAGE_LOWER_THRESHOLD = 0.0001
+export const USD_LOWER_THRESHOLD = 0.009
+
+// APR display thresholds
+export const APR_UPPER_THRESHOLD = 1_000_000
+export const APR_LOWER_THRESHOLD = 0.0000001
+
+// BigNumber processing threshold
+export const BN_LOWER_THRESHOLD = 0.000001
+
+// Fiat formatting threshold
+export const FIAT_CENTS_THRESHOLD = '100000'
```

Comment:
this does not seem to be used? i'd suggest to remove it and the folder

Reply:
removed, thx, tried to centralize